src/dashboard/src/components/archival_storage/views.py

```diff
@@ -306,53 +302,12 @@ def create_aic(request, *args, **kwargs):
         logger.error("Error creating AIC: Form not valid: {}".format(aic_form))
         return redirect('archival_storage_index')
 
-def delete_context(request, uuid):
-    prompt = 'Delete AIP?'
-    cancel_url = reverse("components.archival_storage.views.overview")
-    return RequestContext(request, {'action': 'Delete', 'prompt': prompt, 'cancel_url': cancel_url})
-
-@decorators.confirm_required('delete_request.html', delete_context)
-def aip_delete(request, uuid):
-    try:
-        reason_for_deletion = request.POST.get('reason_for_deletion', '')
 
-        response = storage_service.request_file_deletion(
-           uuid,
-           request.user.id,
-           request.user.email,
-           reason_for_deletion
-        )
+def delete_aip(request, uuid):
+    raise Http404
```

Comment:
What is the tombstone on `delete_aip` for? The view function was renamed and the URL is changed, so the only places I'd expect to call it are from new code.  Same question for `reingest_aip` though the URL & view are unchanged.